src/react/src/components/InitialClaimFlow/ClaimForm/validationSchemas.js

```diff
@@ -22,17 +22,101 @@ export const businessStepSchema = Yup.object().shape({
     businessWebsite: Yup.string().url('Invalid URL'),
 });
 
-// Step 4: Profile validation (all optional fields).
+// Step 4: Profile validation.
 export const profileStepSchema = Yup.object().shape({
+    // Production Location Overview
+    localLanguageName: Yup.string(),
+    officePhoneNumber: Yup.string(),
+    businessWebsite: Yup.string()
+        .nullable()
+        .test('is-valid-url', 'Must be a valid URL', value => {
+            if (!value || value.trim() === '') return true;
+            try {
```

Comment:
I think this check could've been easier. On component level you can use `<TextField type="url">` and for Yup validation schema something like `Yup.string().url('Invalid URL')`